rtris.py

```diff
@@ -4,11 +4,176 @@
 from numpy import add as np_add
 os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
 import pygame,pygame.freetype
+from sys import argv
 
 K_DROP=False
 confpath=os.path.abspath(os.path.expanduser("~/.rtrisconf"))
 
+VERSION="1.0a3"
+COPYRIGHT_YEAR="2019"
+# TODO: do you wanna give out your full name?
+COPYRIGHT_HOLDER="RIEDLER"
+AUTHORS="RIEDLER and Michael Federczuk"
+
+USAGE="usage: %s" % (argv[0])
+HELP="""\
+%s
+    Start Rtris, a Tetris clone written in Python.
+
+    Options:
+      -h, --help      display this summary and exit
+      -V, --version   display version information and exit
+      -d, --debug     print debug information (default)
+      -D, --no-debug  don't print debug information
+
+    Exit Status:
+      (using CommonCodes v1.0.0
+       <https://mfederczuk.github.io/commoncodes/v/1.0.0.html>)
```

Comment:
Python still doesn't work with exit codes

Reply:
What about the `exit` function?  
I'm using them in the `opt_help` and `opt_version_info` functions, for exmaple.  
It worked for me

Reply:
True, but just raising Errors with an appropiate message is easier and can tell you a lot more about the problem.

Reply:
Well that's a simple command line usage error, the message is enough to tell you what went wrong.  
For more complex errors deep inside the code it's better to raise exceptions I agree there.

Reply:
It's always better to raise Exceptions, since consistency is also a very important part of coding. Exit codes are important for libraries, or other parts of code that require to be executed by other code. But for front-end programs it's just better to raise Exceptions.
Also, it's my project please just change it, k?